shared/fs/row/styles.js

```diff
@@ -67,13 +73,105 @@ const rightBox = {
   alignItems: 'center',
 }
 
-export default {
-  rowBox,
-  itemBox,
-  pathItemIcon,
-  pathItemIcon_30,
+const pathItemActionIcon = {
```

Comment:
should use Styles.styleSheetCreate

Reply:
These are actually fed into `styleSheetCreate` below when `rowStyles` is exported.